documentation/book/proc-adding-users-the-strimzi-admin-role.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// assembly-getting-started-strimzi-admin.adoc
+
+[id='proc-adding-users-the-strimzi-admin-role-{context}']
+= Deploying the User Operator using the Cluster Operator
+
+.Prerequisites
+
+* {ProductName} `CustomResourceDefinitions` installed
+
+.Procedure
+
+. Create the `strimzi-admin` cluster role in {ProductPlatformName}.
```

Comment:
Unless "cluster role" is an existing term in OpenShift, I suggest we just say "role".

Reply:
There are two resources, `ClusterRoles` and `Roles`. So I think we need to distinguish it here.